protofsm/state_machine.go

```diff
@@ -356,6 +357,14 @@ func (s *StateMachine[Event, Env]) executeDaemonEvent(
 
 					s.SendEvent(event)
 				})
+
+				if !launched {
+					return errors.New(
```

Comment:
I propose to create a public variable for this error, so it is possible to check errors against it; and reuse it in instances below.

Reply:
What do you mean by "public variable". Like a constant for the error? or like a single name with multiple assignment?